### PR TITLE
chore: Adding protos for KeyExists

### PR DESCRIPTION
**Pull request by mjindalmotiv8**
Adds protocols for a KeyExists API, which can be used to query whether keys exist or not

### CODE REVIEW

File: proto/cacheclient.proto [hunk 1]
```diff
@@ -21,6 +21,7 @@ service Scs {
   rpc Set (_SetRequest) returns (_SetResponse) {}
   rpc SetIfNotExists (_SetIfNotExistsRequest) returns (_SetIfNotExistsResponse) {}
   rpc Delete (_DeleteRequest) returns (_DeleteResponse) {}
+  rpc KeysExists (_KeysExistsRequest) returns (_KeysExistsResponse) {}
```
Comment (kvcache) [inline proto/cacheclient.proto:24]:
```suggestion
  rpc KeysExist (_KeysExistRequest) returns (_KeysExistResponse) {}
```
for it to read a little more naturally (especially for the poor enunciators among us)?

**Review by kvcache — COMMENTED** [review]
looks good, 1 very small question

File: proto/cacheclient.proto (hunk 1 above)
Comment (mjindalmotiv8) [reply proto/cacheclient.proto]: I actually had made this change and updated the doc - did not push the commit yet. Updated just now

**Review by kvcache — APPROVED** [review]
l's gtm

**Pull request merged**